Emby.Server.Implementations/ScheduledTasks/TaskManager.cs

```diff
@@ -49,7 +49,7 @@ public TaskManager(
         /// Gets the list of Scheduled Tasks.
         /// </summary>
         /// <value>The scheduled tasks.</value>
-        public IScheduledTaskWorker[] ScheduledTasks { get; private set; }
+        public ImmutableList<IScheduledTaskWorker> ScheduledTasks { get; private set; }
```

Comment:
We prefer `IReadOnlyList` as its more dynamic with its implementation
```suggestion
        public IReadOnlyList<IScheduledTaskWorker> ScheduledTasks { get; private set; }
```